.gitignore

```diff
@@ -16,29 +17,95 @@
 .idea/
 
 # The .vscode folder contains launch configuration and tasks you configure in
+
 # VS Code which you may wish to be included in version control, so this line
 # is commented out by default.
-#.vscode/
+
+# Visual Studio Code related
+.classpath
+.project
+.settings/
+.vscode/
+
+# packages file containing multi-root paths
+.packages.generated
 
 # Flutter/Dart/Pub related
 **/doc/api/
 **/ios/Flutter/.last_build_id
 .dart_tool/
 .flutter-plugins
 .flutter-plugins-dependencies
+**/generated_plugin_registrant.dart
 .packages
 .pub-cache/
 .pub/
 /build/
+flutter_*.png
+linked_*.ds
+unlinked.ds
+unlinked_spec.ds
 
 # Web related
 lib/generated_plugin_registrant.dart
 
+# Android related
+**/android/**/gradle-wrapper.jar
+**/android/.gradle
+**/android/captures/
+**/android/gradlew
+**/android/gradlew.bat
+**/android/local.properties
+**/android/**/GeneratedPluginRegistrant.java
+**/android/key.properties
+*.jks
+
+# iOS/XCode related
+**/ios/**/*.mode1v3
+**/ios/**/*.mode2v3
+**/ios/**/*.moved-aside
+**/ios/**/*.pbxuser
+**/ios/**/*.perspectivev3
+**/ios/**/*sync/
```

Comment:
add pod lock file for ios also here

Reply:
**/ios/**/Pods/ 
this one in line 75 is for pod lock ig

Reply:
yes this one also, and in the root folder of ios there is a Podfile.lock file 

<img width="288" alt="Screenshot 2021-03-30 at 10 13 34 AM" src="https://user-images.githubusercontent.com/21235425/112935042-97769e80-9140-11eb-8504-c9c261fa7365.png">

Reply:
hey @hridhi any update on the above, its just a small change and then we are good for merge.

Reply:
sorry for delay please check